chain/network/src/types.rs

```diff
@@ -838,12 +838,22 @@ pub enum NetworkRequests {
 }
 
 pub struct ValidateEdgeList {
+    /// List of Edges, which will be sent to `EdgeVerifierActor`.
```

Comment:
I think we're calling it 'validate' now, right? (update here and in other places in the comments below)

Reply:
Yes,  fixed this after rabase.